vip_hci/leastsq/leastsq.py

```diff
@@ -215,7 +214,7 @@ def _leastsq_patch(array, nseg, indices, angles, pa_threshold, metric,
     matrix_res = np.zeros((values.shape[0], yy.shape[0]))
     for i in range(n_frames):
         vector = pn.DataFrame(mat_dists_ann[i])
-        vector.columns = ['i']
+        vector.columns = [i]
```

Comment:
This is not slicing using the loop variable i, the quotes are needed

Reply:
What exactly is the purpose of setting the label of the column to `'i'`? I thought it is for debugging, as the labels show when displaying the DataFrame in jupyter.

Reply:
To be honest I don't remember why I named the column, perhaps the line:
vector.columns = ['i']
is not needed in the end

Reply:
removed in [commit fee553c](https://github.com/vortex-exoplanet/VIP/pull/149/commits/fee553ceded56f1edc9c6d49dd2b5fe8e4e2faff)

Reply:
FYI, in `frdiff/framediff.py:225`, you have written
``` python
df = pn.DataFrame(ind)  # sorting using pandas dataframe
df.columns = ['i', 'j']
df = df.sort_values('i')
```

so the column labels are used for sorting! But in this case, no sorting was done.